packages/create-toolpad-app/src/index.ts

```diff
@@ -161,6 +161,323 @@ const scaffoldProject = async (absolutePath: string, installFlag: boolean): Prom
   }
 };
 
+const scaffoldCoreProject = async (absolutePath: string): Promise<void> => {
+  // eslint-disable-next-line no-console
+  console.log();
+  // eslint-disable-next-line no-console
+  console.log(
+    `${chalk.blue('info')} - Creating Toolpad Core project in ${chalk.blue(absolutePath)}`,
+  );
+  // eslint-disable-next-line no-console
+  console.log();
+
+  const packageJson: PackageJson = {
```

Comment:
Jan and I did discuss this and felt like we could eventually move to this structure, but begin with strings for simplicity and efficiency

Reply:
If the idea is that in the future we will do things like pre-configure different auth providers, we're going to need to have some templating mechanism. I believe this mechanism will be much easier to write and maintain when it's just string composition in functions, rather than try to adjust pre-existing file content from a template file.

It'll also make it easier to do things like re-using the templating logic to generate codesandboxes in the browser.